File: src/question/month08/day0811/ms_0811.cpp
#include <iostream>

using namespace std;

int main() {
	int data[10];

	for (int i = 0; i < sizeof(data) / sizeof(data[0]); i++) {
		scanf_s("%d", &data[i]);
	}

	for (int i = 0; i < sizeof(data) / sizeof(data[0]) - 1; i++) {
		int half = i;

		for (int j = i + 1; j < sizeof(data) / sizeof(data[0]); j++) {
			if (data[half] < data[j])
				half = j;

			int temp = data[i];
			data[i] = data[half];
			data[half] = temp;
		}
	}

	for (int i = 0; i < sizeof(data) / sizeof(data[0]); i++) {
		cout << data[i] << " ";
	}

	return 0;
}

File: src/question/month08/day0811/ms_0811.cs
using System;

namespace algorithm_32
{
    class ms_0811
    {
        static void Main(string[] args)
        {
            int[] data = new int[10];

            for(int i = 0; i < data.Length; i++) {
                data[i] = Convert.ToInt32(Console.ReadLine());
            }

            for(int i = 0; i < data.Length - 1; i++) {
                int half = i;
                
                for(int j = i+1; j < data.Length; j++) {
                    if(data[half] < data[j]) {
                        half = j;
                    }

                    int temp = data[i];
                    data[i] = data[half];
                    data[half] = temp;
                }
            }

            foreach(int value in data) {
                Console.Write($"{value} ");
            }
        }
    }
}
